core/src/main/java/nl/nn/adapterframework/stream/MessageContext.java

```diff
@@ -75,11 +76,20 @@ public MessageContext withMimeType(String mimeType) {
 		try {
 			withMimeType(MimeType.valueOf(mimeType));
 		} catch (InvalidMimeTypeException imte) {
+			String parsed = StringUtil.splitToStream(mimeType, ";").filter(e -> !e.contains("=")).findFirst().orElse(null);
+			if(parsed != null) {
+				try {
+					return withMimeType(MimeType.valueOf(parsed));
+				} catch (InvalidMimeTypeException imte2) {
+					LOG.debug("tried to parse cleansed mimetype [{}]", parsed, imte2);
+				}
+			}
 			LOG.warn("unable to parse mimetype from string [{}]", mimeType, imte);
```

Comment:
Willen we deze case wel nog steeds op level "WARN" blijven loggen, terwijl we daarboven ook al met level "DEBUG" loggen?
Dan krijgen we 2 stacktraces achter elkaar.